tests/FSharp.Collections.ParallelSeq.Tests/FSharp.Collections.ParallelSeq.Tests.fsproj

```diff
@@ -15,6 +14,7 @@
       <Private>True</Private>
     </ProjectReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta7" />
```

Comment:
Just out of curiosity, where do you use that/what is it for?

Reply:
`NUnit` uses it and without this package Visual Studio fails to discover unit tests. It was not referenced explicitly by NUnit because it is beta. If we are not interested in debugging of unit tests from Visual Studio we can revert changes from `FSharp.Collections.ParallelSeq.Tests.fsproj`.

Reply:
No I am fine with it. Was just curious.